app/src/main/java/net/gsantner/markor/util/DocumentIO.java

```diff
@@ -127,8 +130,8 @@ public static synchronized Document loadDocument(Context context, Bundle argumen
                 document.setFormat(TextFormat.FORMAT_KEYVALUE);
             } else if (TextFormat.CONVERTER_MARKDOWN.isFileOutOfThisFormat(fnlower)) {
                 document.setFormat(TextFormat.FORMAT_MARKDOWN);
-            } else if (fnlower.endsWith(".txt") || fnlower.endsWith(".zim")) {
-                document.setFormat(TextFormat.FORMAT_PLAIN);
+            } else if (isZimWikiFile(fnlower, document)) {
```

Comment:
This was a not-so-easy design decision I made:
The problem is that for zim wiki text files, it's not enough to look at the filename. Instead, the content has to be examined to find out if the zim header is present there or not.
`TextFormat.CONVERTER_ZIMWIKI.isFileOutOfThisFormat` only gets the filepath as an argument. For all other formats, checking the ending is enough, but for zim not. So in order to use the method, the file has to be opened and checked. However, I found out that the passed name is not necessarily the full file path, then causing an error when attempting to open it. 
So instead of digging deeper in the base code and making potentially harmful changes in the existing code, I decided to just introduce the `isZimWikiFile` method. Advantage here: the content is readily available in the `Document` (also better performance-wise) and can easily be checked for the header.